src/main/features/certificateOfService/routes/file-upload.ts

```diff
@@ -15,12 +15,18 @@ export default express.Router()
   .post(Paths.fileUploadPage.uri,
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
       const form = new formidable.IncomingForm()
+      const user: User = res.locals.user
       const FILE_SIZE_LIMIT: number = 10485760
       form.keepExtensions = true
+      req.setTimeout(7200000,function () {
```

Comment:
Extract this magic number to a constant with a name that makes it clear what the timeout value is.